src/os_auth/main-client.c

```diff
@@ -133,7 +128,7 @@ int main(int argc, char **argv)
                 if (!optarg) {
                     merror_exit("-g needs an argument");
                 }
-                dir = optarg;
+                // dir = optarg; (NEVER USED)
```

Comment:
If this option is no longer used, we can add the following message: mwarn(DEPRECATED_OPTION_WARN,"-D");